src/data/markdown/docs/30 xk6-browser/01 Get started/03 Running xk6-browser.md

```diff
@@ -235,3 +236,50 @@ export function news() {
 The preceding code contains two scenarios. One for the browser-level test called `browser` and one for the protocol-level test called `news`. Both scenarios are using the [constant-vus executor](/using-k6/scenarios/executors/constant-vus/) which introduces a constant number of virtual users to execute as many iterations as possible for a specified amount of time. 
 
 Since it's all in one script, this allows for greater collaboration amongst teams.
+
+## Run xk6-browser tests in a Docker container
+
+Apart from running the tests locally, you can also run your xk6-browser scripts in a Docker container using Docker Compose by creating the following Dockerfile and docker-compose file:
+
+<CodeGroup labels={["Dockerfile", "docker-compose.yaml"]} lineNumbers={[true]}>
+
+```bash
```

Comment:
Is there anyway of linking to the files in the github repo instead of adding them here? Or maybe add a link too?

Just want mitigate the risk of forgetting to update the files here if we update them in the repo.

Reply:
You are right! I have changed it to links instead :)